parasolr/query.py

```diff
@@ -159,13 +160,68 @@ def get_facets(self) -> Dict[str, int]:
         return self.solr.query(**query_opts).facet_counts['facet_fields']
 
     @staticmethod
-    def _lookup_to_filter(key, value) -> str:
-        """Convert keyword argument key=value pair into a Solr filter.
-        Currently only supports simple case of field:value."""
+    def _lookup_to_filter(key: str, value: Any) -> str:
+        """Convert keyword/value argument, with optional filters, into a
+        Solr query. Currently supports simple key-value pairs, searching for
+        empty/unset values, and __in filters of lists.
+
+            Returns: A propertly formatted Solr query string.
+        """
+
+        # Flag if a null/empty search is being conducted
+        null_filter = False
```

Comment:
The reason, in this instance, for a general case, was to support a lookup value that was `'' or None` in a filter search and to avoid having to fiddle with the logic for it separately in each instance. It would be easy enough to do separate logic for something like `__exists` or `__isnull` as you suggest below, and that might simplify this.

Reply:
I like `exists` for a generic term to use for this.